### ISSUE TITLE
Hw08

### ISSUE THREAD

**Issue by sinaneza** [STAT545-UBC/Discussion#430]
Dear Professor and TAs

Since during these last days of this semester our schedule has become so congested, I would appreciate whether you could extend the due date of homework No. 8
@jennybc

**Comment by jennybc** [comment]
YES I am happy to extend the due date for Homework 8 from Friday Nov 25 to Monday 28.

**Comment by jennybc** [comment]
@samhinshaw Would you be willing to change that in the appropriate places within the course website?
